### DISCUSSION TITLE
rspamd check vs milter (rspamd) check

### DISCUSSION THREAD

**Discussion by mmatous** [emersion/maddy#624] [General]
Hi, is there any difference between communicating with rspamd via milter protocol and `rspamd` directive in maddy config file? Aside from `rspamd` not supporting unix domain sockets.

**Comment by foxcpp** [comment]
There is not really much of a difference now. `check.rspamd` may be a little more efficient due to connection pooling and so on, but that's it.
`check.rspamd` was implemented before `check.milter` implemented protocol version required by rspamd.

**Comment by mmatous** [comment]
Thanks! `rspamd` might even be worth dropping now, since it's less featureful 300 LoC alias to `milter` :+1:
